config.json

```diff
@@ -33,7 +33,8 @@
       "customized_checkout",
       "account_payment_methods",
```

Comment:
Please delete `account_payment_methods` feature from the features list as it's not needed for this new version. It causes an un-needed resource to be created and returned with failing validation that is related to the old cornerstone version.

Reply:
fixed